src/infrastructure/organisations/api.js

```diff
@@ -10,16 +11,12 @@ const getUserOrganisations = async (userId, correlationId) => {
   return client.get(`/organisations/associated-with-user/${userId}`, correlationId);
 };
 
-const getUserOrganisationsV2 = async (userId, correlationId) => {
-  return client.get(`/organisations/v2/associated-with-user/${userId}`, correlationId);
-};
+const getUserOrganisationsV2 = async (userId) => organisation.getOrganisationsForUserIncludingServices(userId);
```

Comment:
Hi @shivanshuit914 hope all tests are fine, I mean the existing tests

Reply:
Yes they all are passing

```
Test Suites: 12 passed, 12 total
Tests:       57 passed, 57 total
Snapshots:   0 total
Time:        3.743s, estimated 8s
```